### PR TITLE
change compose form placeholder after user clicked content wanrning button

### PR DESCRIPTION
**Pull request by walf443**
reading https://github.com/tootsuite/mastodon/pull/2305, I thinks it's difficult for beginner user to imagine what should write to spoiler placeholder field.

So how about like this?

Before Click CW Button:

before_cw

After Click CW Button:

mastodon

I thinks it's not best message of it. Suggestions are welcome!!

### CODE REVIEW

**Comment by yumetodo** [comment]
found reaction about this PR
https://ostatus.yoh2.ddo.jp/users/yoh2/updates/45

It is same as my opinion pointed out in #2310 that writing message to explain about the input area instead of writing example message.

File: app/assets/javascripts/components/features/compose/components/compose_form.jsx [hunk 1]
```diff
@@ -18,7 +18,8 @@ import TextIconButton from './text_icon_button';
 
 const messages = defineMessages({
   placeholder: { id: 'compose_form.placeholder', defaultMessage: 'What is on your mind?' },
-  spoiler_placeholder: { id: 'compose_form.spoiler_placeholder', defaultMessage: 'Content warning' },
+  hide_text_placeholder: { id: 'compose_form.hide_text_placeholder', defaultMessage: 'This text hide by default.' },
```
Comment (yiskah) [inline app/assets/javascripts/components/features/compose/components/compose_form.jsx]: This phrasing is a bit off for English, but a good try. I'd go for something like "Text to hide behind warning"

File: app/assets/javascripts/components/features/compose/components/compose_form.jsx [hunk 2]
```diff
@@ -18,7 +18,8 @@ import TextIconButton from './text_icon_button';
...
+  spoiler_placeholder: { id: 'compose_form.spoiler_placeholder', defaultMessage: 'Write warnings before hide text.' },
```
Comment (yiskah) [inline app/assets/javascripts/components/features/compose/components/compose_form.jsx]: Minor change, for English should be "Warning before hidden text" or "Write warning before hidden text"

File: app/assets/javascripts/components/locales/en.jsx [hunk 3]
```diff
@@ -34,12 +34,13 @@ const en = {
...
   "column.public": "Federated timeline",
+  "compose_form.hide_text_placeholder": "Hide text behind warning",
   "compose_form.placeholder": "What is on your mind?",
   "compose_form.privacy_disclaimer": "Your private status will be delivered to mentioned users on {domains}. Do you trust {domainsCount, plural, one {that server} other {those servers}}? Post privacy only works on Mastodon instances. If {domains} {domainsCount, plural, one {is not a Mastodon instance} other {are not Mastodon instances}}, there will be no indication that your post is private, and it may be boosted or otherwise made visible to unintended recipients.",
   "compose_form.publish": "Toot",
   "compose_form.sensitive": "Mark media as sensitive",
-  "compose_form.spoiler_placeholder": "Content warning",
-  "compose_form.spoiler": "Hide text behind warning",
+  "compose_form.spoiler": "Hide text behind warning.",
+  "compose_form.spoiler_placeholder": "Write warnings before hide text.",
```
Comment (yiskah) [inline app/assets/javascripts/components/locales/en.jsx]: Another spot. `hide` should be `hidden`

File: app/assets/javascripts/components/locales/fr.jsx [hunk 4]
```diff
@@ -36,7 +36,6 @@ const fr = {
   "compose_form.publish": "Pouet",
   "compose_form.sensitive": "Marquer le média comme délicat",
   "compose_form.spoiler": "Masquer le texte derrière un avertissement",
```
Comment (yiskah) [inline app/assets/javascripts/components/locales/fr.jsx]: Is there a reason you are removing this line from the French locale?

**Review by yiskah — CHANGES_REQUESTED** [review]
There are some minor English grammatical errors. When you `hide` text, it is `hidden`. I'm unsure why you are removing lines from all the locales

File: app/assets/javascripts/components/locales/fr.jsx (hunk 4 above)
Comment (walf443) [inline app/assets/javascripts/components/locales/fr.jsx]: I changed compose_form.spoiler_placeholder meaning. so translation should change.

**Comment by beatrix-bitrot** [comment]
the branch for this PR was deleted 16 days ago, should it be closed?

**Comment by walf443** [comment]
Thanks.

**Pull request closed without merging**